src/manifest.ts

```diff
@@ -9,3 +9,33 @@ export interface Moziot<T> {
   schema: Record<string, unknown>;
   config: T;
 }
+
+export interface ManifestFile {
+  manifest_version: number;
+  id: string;
+  name: string;
+  short_name: string;
+  version: string;
+  description: string;
+  homepage_url: string;
+  license: string;
+  author: string;
+  gateway_specific_settings: GatewaySpecificSettings;
+  options: Options;
+}
+
+export interface GatewaySpecificSettings {
+  webthings: WebthingsSettings;
+}
+
+export interface WebthingsSettings {
+  exec: string;
```

Comment:
`exec` is optional, since it's not required for pure extensions.

Reply:
done